src/_sass/_related.scss

```diff
@@ -0,0 +1,77 @@
+@use "./breakpoints";
+@use "./colours";
+@use "./typography";
+
+.related {
+  width: 100%;
+  padding: 1em;
+  background-color: colours.$accent-a;
```

Comment:
Is this an intentional background colour change? Previously it was `colours.$cream`

Reply:
Is it a hangover from splitting out the main navigation and page outline sidebars?